content/direct-deposit-management/release-notes/2024-11-27.md

```diff
@@ -0,0 +1,3 @@
+We added an alert notification for direct deposit accounts flagged for “potential” fraud. When a backend VA data source has an account flagged for potential fraud activity, the API response includes an HTTP XXX alert message.
```

Comment:
Is XXX a code number?

Reply:
This code number is still TBD but we'll have an answer tomorrow. It could be 200 or 422 pending a discussion.